packages/linux/package.mk

```diff
@@ -16,6 +16,12 @@ PKG_STAMP="$KERNEL_TARGET $KERNEL_MAKE_EXTRACMD"
 PKG_PATCH_DIRS="$LINUX"
 
 case "$LINUX" in
+  amlogic-mainline)
+    PKG_VERSION="df28223b151155ab0edd8419b0347a7135443fd3" # linux-5.1-le-amlogic-v2
```

Comment:
I'd rather just move it to the mainline kernel and have a patch dir for amlogic

Reply:
I thought you did not want a few thousands lines long patch. ;)

Sure, this can be done and patch will become smaller as majority of patches are already in 5.2 or staged for 5.3.

Reply:
the whole point of moving to mainline is to use mainline as the base 😜

Reply:
Patches would have to be moved to `project/Amlogic/patches` because `PKG_PATCH_DIRS="default"` for `*)` case. Is that OK? This would match Allwinner project.

Reply:
> Patches would have to be moved to `project/Amlogic/patches` because `PKG_PATCH_DIRS="default"` for `*)` case. Is that OK? This would match Allwinner project.

yes